modules/web/route.go

```diff
@@ -18,16 +17,9 @@ import (
 )
 
 // Bind binding an obj to a handler
-func Bind(obj interface{}) http.HandlerFunc {
-	tp := reflect.TypeOf(obj)
-	if tp.Kind() == reflect.Ptr {
-		tp = tp.Elem()
-	}
-	if tp.Kind() != reflect.Struct {
-		panic("Only structs are allowed to bind")
-	}
+func Bind[T any](obj T) http.HandlerFunc {
```

Comment:
Why remove the check for `Only structs are allowed to bind`?

Reply:
There is no method to check that in generics function except we still use `reflect`. But I think that will not be what we want to do.